pkg/docker/docker.go

```diff
@@ -54,6 +65,9 @@ const (
 	// DefaultDockerTimeout specifies a timeout for Docker API calls. When this
 	// timeout is reached, certain Docker API calls might error out.
 	DefaultDockerTimeout = 20 * time.Second
+
+	// DefaultShmSize  is the default ShmSize to use (in bytes) if not specified.
```

Comment:
there's an extra space here and should probably spell our "shared memory size" in the description (at least i'm assuming that's what this value is)